### DISCUSSION TITLE
Feature: Date Picker with calendar weeks (optionally shown in first column)

### DISCUSSION THREAD

**Discussion by d-m-s** [vmware/clarity#6369] [Ideas]
see Feature Request: https://github.com/vmware/clarity/issues/5273 Date Picker with calendar weeks (optionally shown in first column) 

Unfortunately I haven't found any public available feature request list, ... **is this still somehow in focus for development?**

We would require this feature for one of our applications and are looking for the best way to provide it!

 - I guess implementation in the angular components, would not be recommended 
 as they're pretty much feature-freezed, am I right?
 - Is the Datepicker available as CORE component any time soon? 
 Could we somehow contribute? 

Please point us in the most efficient direction on how we could realize the feature.

Thanks!

**Comment by bbogdanov** [comment]
Hi @d-m-s,

The feature request is on the right place here. 

The `@clr/angular` is frozen for features, yes. 

The DatePicker in Core is available https://clarity.design/storybook/core/?path=/story/forms-date--page 

As Jeremy mentioned in the issue you linked the feature is captured in our backlog and we are looking forward to provide this feature. We are small team and it will take some time until we get to it.

We are always happy to collaborate with the community so if you are interested to contribute let us know and we can get in touch and discuss the details.
